README.rst

```diff
@@ -1,7 +1,44 @@
 Byterun
 -------
 
-This is a pure-Python implementation of a Python bytecode execution virtual
-machine.  I started it to get a better understanding of bytecodes so I could
-fix branch coverage bugs in coverage.py.
+
+
+Byterun is a python interpreter written in just 1451 lines of Python.
+Byterun's goal is to clearly explain Python's design. 
```

Comment:
Byterun is more targeted at the CPython bytecode scheme than Python as a language.